src/fetch/period_slippage.py

```diff
@@ -19,6 +19,22 @@
 logging.config.fileConfig(fname="logging.conf", disable_existing_loggers=False)
 
 
+SELECT_UNUSUAL_SLIPPAGE = """
+select
+    block_time,
+    rpt.solver_name,
+    concat('0x', encode(rpt.tx_hash, 'hex')) as tx_hash,
+    usd_value,
+    batch_value,
+    100 * usd_value / batch_value as relative_slippage
+from results_per_tx rpt
+join gnosis_protocol_v2."batches" b
+    on rpt.tx_hash = b.tx_hash
+where -1 * usd_value > '{{SignificantValue}}'
```

Comment:
Do you think it makes sense to work with abs()? I think it would be nice if we also get an alert with too much positive slipapge

Reply:
Yea sure, I think that is a great idea!

Reply:
Just Be aware that this might be a bit more noisy than we are hoping for:

Here is [last week's results](https://dune.com/queries/645559):
![Screenshot 2022-06-20 at 3 25 37 PM](https://user-images.githubusercontent.com/11778116/174611817-9a78e62e-90d5-4cd5-b84b-7f6808189c86.png)


Here are this weeks:

![Uploading Screenshot 2022-06-20 at 3.28.25 PM.png…]()

Reply:
Good enough. I will update with the fix soon.